### ISSUE TITLE
host processing latency spikes 

### ISSUE THREAD

**Issue by AmplificPanda** [ClassicOldSong/Apollo#120]
Randomly getting high host processing latency... the middle number spikes to 40ms every 5 seconds. 

No idea what could be causing this?

game: Horizon Zero Dawn Remastered on Steam 

configuration: 
windows 11 pro
rtx 4070ti 
5600x 
32gb ddr4 
resolution 4K @60hz on client using Artemis on NVIDIA Shield 4K Pro 2019
aorus b550 aorus elite ax v2 
hardwired by ethernet USB 3.0 To Ethernet adapter (using this because my motherboards Ethernet port might have hardware packet loss) 
encoder: NVENC

**Comment by ClassicOldSong** [comment]
Only this game is affected or all games are affected? Is steam recording turned on?

**Comment by ClassicOldSong** [comment]
I'm using 4 dimms of DDR4 with XMP also, but I got voltage turned down to 1.29V and running fine with 3200Mhz, even faster than the default XMP profile.

Is 1.4V too high? I have bit fuse on all four ram sticks after 5 years of default XMP profile (1.35V 3000Mhz) so I replaced them with my backup storage and lowered over clocking voltage, tested stable with multiple runs of memtset86+, now it's running smoothly without issues.
